website/docs/r/secrets_sync_gh_apps_destination.html.md

```diff
@@ -0,0 +1,69 @@
+---
+layout: "vault"
+page_title: "Vault: vault_secrets_sync_github_apps_destination resource"
+sidebar_current: "docs-vault-resource-secrets-sync-github-apps-destination"
+description: |-
+Creates a GitHub App to synchronize secrets in Vault
+---
+
+# vault\_secrets\_sync\_github\_apps\_destination
+
+Creates a GitHub App to synchronize secrets in Vault. Requires Vault 1.16+.
+*Available only for Vault Enterprise*.
+
+~> **Important** All data provided in the resource configuration will be
+written in cleartext to state and plan files generated by Terraform, and
+will appear in the console output when Terraform runs. Protect these
+artifacts accordingly. See
+[the main provider documentation](../index.html)
+for more details.
+
+For more information on syncing secrets with GitHub, please refer to the Vault
+[documentation](https://developer.hashicorp.com/vault/docs/sync/github).
+
+## Example Usage
+
+```hcl
+resource "vault_secrets_sync_github_apps_destination" "github-apps" {
+  name                 = "gh-apps-dest"
+  app_id               = var.app_id
+  private_key          = file(var.privatekey_file)
+  secret_name_template = "vault_{{ .MountAccessor | lowercase }}_{{ .SecretPath | lowercase }}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `namespace` - (Optional) The namespace to provision the resource in.
+  The value should not contain leading or trailing forward slashes.
+  The `namespace` is always relative to the provider's configured [namespace](/docs/providers/vault#namespace).
+
+* `name` - (Required) Unique name of the GitHub destination.
+
+* `app_id` - (Required) The ID of the GitHub app the user created on GitHub. Can be modified.
+  Can be omitted and directly provided to Vault using the `GITHUB_APPS_PRIVATE_KEY` environment
+  variable.
+
+* `repository_owner` - (Optional) The content of a PEM formatted private key generated on GitHub for the app. 
+  If provided inline, the line feeds must be properly encoded.
+  Can be omitted and directly provided to Vault using the `GITHUB_APPS_ID` environment
+  variable.
+
+* `secret_name_template` - (Optional) Template describing how to generate external secret names.
+  Supports a subset of the Go Template syntax.
```

Comment:
```suggestion
```

This doesn't apply to the github app configuration, since it's not a destination.